### ISSUE TITLE
Appeal system possibly cannot automatically identify IPv6 block on IPv4 address

### ISSUE THREAD

**Issue by scarletcs** [UTRS2/utrs#745]
I am currently blocked on Wikipedia under the IPv6 range `‪2A09:BAC1:0:0:0:0:0:0/32`.

The appeal system's IP checker (https://api64.ipify.org/) reports my IP as an IPv4 address.

Although mainsite Wikipedia associates my IP with a block, the appeal system appears to be unable to perform the same association and my appeal is automatically placed under a NOTFOUND status.

I've opened a separate issue related to another bug in this same appeal, #744. That might be completely unrelated or it might also relate to the inability to associate this IP with that block. (I promise I'm not filing two issues to try to speed up my appeal, I've just noticed two separate issues going on.)

---

Pictured: my block.

![wiki block](https://github.com/UTRS2/utrs/assets/4585677/e5070536-8b56-45e8-9bcf-becfbb4ee8e3)

Pictured: my appeal. The screened IP is IPv4.

![wiki appeal](https://github.com/UTRS2/utrs/assets/4585677/30c42512-74e5-40be-bb17-32c42d0bdfa0)

**Comment by scarletcs** [comment]
As mentioned in #744 this may be specifically because the IPv6 I am blocked under is not a valid IPv6 (and yet I am blocked under it somehow). I'm going to close this issue since it's probably just the one root problem.

dupe of #744
